AlttpRandomizer/Random/Randomizer.cs

```diff
@@ -75,7 +75,8 @@ public string CreateRom(RandomizerOptions options)
             {
                 if (options.Filename.Contains("\\") && !Directory.Exists(options.Filename.Substring(0, options.Filename.LastIndexOf('\\'))))
                 {
-                    Directory.CreateDirectory(options.Filename.Substring(0, options.Filename.LastIndexOf('\\')));
+                    String filePath = FileName.Fix(options.Filename, string.Format(romLocations.SeedFileString, seed), complexity);

```

Comment:
Seems like it would be better to use FileName.Fix on the directory before checking to see if it exists instead of after.